packages/databases-collections-list/src/items-grid.tsx

```diff
@@ -108,7 +108,7 @@ const GridControls = () => {
 
   return (
     <>
-      {createControls && <div className={control}>{createControls}</div>}
+      {createControls && <div className={control} data-testid="create-controls">{createControls}</div>}
```

Comment:
Putting data-testid on containers like these was the best I could come up with because I can't see how to add anything specific to leafygreen buttons.